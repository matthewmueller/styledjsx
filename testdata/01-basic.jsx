export default () => (
  <main class="main">
    <h1>hello</h1>
    <style scoped>{`
      .main {
        background: blue;
      }
      h1 {
        color: red;
      }
    `}</style>
  </main>
)
