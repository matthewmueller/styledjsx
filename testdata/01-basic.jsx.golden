import Style from "styled-jsx";
export default () => (
  <main class="jsx-8mUTT main">
    <h1 class="jsx-8mUTT">hello</h1>
    <Style scoped id="jsx-8mUTT">{`.main.jsx-8mUTT{background:#00f}h1.jsx-8mUTT{color:red}`}</Style>
  </main>
)
